sys/include/net/ieee802154.h

```diff
@@ -114,6 +114,11 @@ extern "C" {
  */
 #define IEEE802154_SIFS_MAX_FRAME_SIZE  (18U)
 
+/**
+ * @brief ACK Timeout period in symbols
+ */
+#define IEEE802154_ACK_TIMEOUT_SYMS     (54)
```

Comment:
Want to sneak in a commit to also use this in `at86rf215` instead of `AT86RF215_ACK_PERIOD_IN_SYMBOLS`

Reply:
done!